lib/resilient/circuit_breaker.rb

```diff
@@ -17,9 +17,10 @@ def self.reset
     # registered. If key does exist, it returns registered instance instead of
     # allocating a new instance in order to ensure that state/metrics are the
     # same per key.
-    def self.get(key: nil, open: false, properties: nil, metrics: nil, registry: nil)
+    def self.get(key: nil, properties: nil, metrics: nil, registry: nil)
+      key = Key.wrap(key) unless key.nil?
```

Comment:
Fixed by 3b87ecab26fb4fcb5d322db9e20ccca87cecc391. 

I also considered dropping the kwarg for key and doing CircuitBreaker.get(key, properties: ..., etc) or even just dropping the kwargs completely and using them in order (key, properties, metrics), but it wasn't super obvious if that was in improvement so I left it the same.